community-content/pytorch_pre_built_images_deployment/pytorch-text-classification-vertex-ai-train-tune-deploy.ipynb

```diff
@@ -0,0 +1,4106 @@
+{
```

Comment:
<p>This is a very comprehensive doc, which is good in some ways; it covers more use cases for users to get started with.</p><p> </p><p>On the other hand, it also presents itself as complex and daunting and could scare customers away.</p><p> </p><p>I wonder if we could hit a sweet spot by exemplifying one method for training and then moving the rest to their own notebook and pointing to those.</p><p> </p><p>For PyTorch serving release I would think local training would be the quickest way to demonstrate results. I could be convinced we should do cloud training, but that section is quite long, too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>+1. Let's focus on what we mainly want to present in this notebook, ie model deployment. My suggestion would be to skip the training part altogether. Can we take a pretrained model and show how to deploy that ?</p>

Reply:
Thanks! Removed the training parts from this notebook.